app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
+    @posts = @user.posts
```

Comment:
## Required Changes ♻️
- [ ] Your app does not solves N+1 problem when fetching all posts and their comments for a user. An example of this in a `posts#index` action should be:
`@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts` 

Kindly ensure to improve upon the same.

Reply:
Yes, it does. If I do it the way you are telling me, this error is pointed by the bullet.
![image](https://user-images.githubusercontent.com/47539402/200860981-4e794de1-8a9e-4f68-a251-b62f7f56bd1b.png)
It was giving me errors like this before. The bullet was asking me to remove the ones that were in the wrong places and add the ones in the right places. So I improved the app following the bullet indications.  I was also checking if the app was doing fewer queries in the process to make sure that was working correctly.